.github/workflows/build.yml

```diff
@@ -16,29 +16,16 @@ jobs:
         if: github.repository == 'pimcore/docker'
         strategy:
             matrix:
-                php: [ '7.4', '8.0', '8.1' ]
-                variant: [ cli, fpm ]
-                distro: [ bullseye, buster ]
-                debug: [ debug, no-debug ]
-                exclude:
-                    -   php: '7.4'
-                        distro: bullseye
-                    -   php: '8.0'
-                        distro: buster
-                    -   php: '8.1'
-                        distro: buster
+                php: [ '8.0', '8.1' ]
+                distro: [ bullseye ]
+                target: [ pimcore_php, pimcore_php_debug, pimcore_php_supervisord ]
```

Comment:
There's still a lot of `${{ matrix.debug }}` usages left, that needs to be replaced, by ${{ matrix.target .... }}